guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<ol><li>The legend/label of this figure can be improved. Maybe put them on the top or at the bottom</li><li>Consider numbering figures. For example, Figure 1. Image comparing FCN and PSPNet [1]. Same thing for the rest of guide</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<ol><li>How can I do that? The label is at the bottom and the legend is on the right. This image is directly from the paper. It seems to be a good for me.</li><li>Since I am not using figure names in the text, is it still important to add the figure numbers?</li></ol>

Reply:
1. If it's hard to change, that's fine.
2. Yes, adding figure number is considered a good convention.

Reply:
Left 1. Did 2.
Thanks.